documentation/openapi.json

```diff
@@ -351,7 +357,8 @@
             "in": "query",
             "description": "The profiles enabled for the trace session.",
             "schema": {
-              "$ref": "#/components/schemas/TraceProfile"
```

Comment:
This doesn't look right to me. The parameter type is an enum. The parameter schema in OpenAPI shouldn't be just a `string` with a default value.